tests/unit-fail-compilation/dup_params1.ref

```diff
@@ -1,7 +1,7 @@
 File: dup_params1.c
 Line: 1
-Error: CV-FD1
-Description: duplicate parameters in function definition.
+Error: CV-TDL6
```

Comment:
I believe you will also need to update CV-FD1 in the examples/simple directory. Do you know if it's even possible to trigger this error anymore or has it been completely superceded?